src/js/tabs/configuration.js

```diff
@@ -21,7 +21,11 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
     }
 
     function load_config() {
-        MSP.send_message(MSPCodes.MSP_FEATURE_CONFIG, false, false, load_beeper_config);
+        MSP.send_message(MSPCodes.MSP_FEATURE_CONFIG, false, false, load_mixer_config);
+    }
+
+    function load_mixer_config() {
```

Comment:
This tab already loads the mixer config: https://github.com/betaflight/betaflight-configurator/pull/1121/files#diff-3e20577ee842fe689a9e108c80b9bd8fR52